src/Microsoft.AspNetCore.Mvc.Abstractions/project.json

```diff
@@ -1,5 +1,9 @@
 {
-  "description": "The core abstractions of ASP.NET MVC.",
+  "description": "ASP.NET Core MVC abstractions and interfaces for action invocation and dispatching, authorization, action filters, formatters, model binding, routing, validation, and more.",
```

Comment:
Should probably mention common types such as `IActionResult`, `ModelStateDictionary`, ...